lib/src/config/unit_tests.rs

```diff
@@ -76,18 +85,60 @@ fn fastly_toml_files_with_simple_backend_configurations_can_be_read() {
     );
 }
 
-/// Unit tests for the `testing` section of a `fastly.toml` package manifest.
+/// Show that we can successfully parse a `fastly.toml` with local_server.dictionaries configurations.
+///
+/// This provides an example `fastly.toml` file including a `#[local_server.dictionaries]` section.
+#[test]
+fn fastly_toml_files_with_simple_dictionary_configurations_can_be_read() {
+    let dir = tempdir().unwrap();
+
+    let file_path = dir.path().join("a.json");
+    let mut file = File::create(&file_path).unwrap();
+    writeln!(file, "{{}}").unwrap();
+    let config = FastlyConfig::from_str(format!(
+        r#"
+            manifest_version = "1.2.3"
+            name = "dictionary-config-example"
+            description = "a toml example with dictionary configuration"
+            authors = [
+                "Amelia Watson <awatson@fastly.com>",
+                "Inugami Korone <kinugami@fastly.com>",
+            ]
+            language = "rust"
+
+            [local_server]
+                [local_server.dictionaries]
+                    [local_server.dictionaries.a]
+                    file="{}"
```

Comment:
oh! i forgot one final detail in my last round of comments.

it would be nice if we were able to future-proof this section, leaving open the possibility of other dictionary formats e.g., defining the dictionary inline inside of the `fastly.toml` itself. (_note: that is absolutely something that we can address another day, since this is already a hefty pr._)

would you mind adding a `format = "json"` field here, and to the `Dictionary` struct? if this is anything _but_ `"json"` we can exit with an error until we introduce support for other formats later.

Reply:
That's a great idea, I had a go at this in [`72f860e` (#61)](https://github.com/fastly/Viceroy/pull/61/commits/72f860eb751bd70658e479e3e4d205eeb6020c1f)